core/trino-main/src/test/java/io/trino/sql/planner/iterative/rule/test/PlanBuilder.java

```diff
@@ -667,6 +668,12 @@ public TableScanBuilder setUpdateTarget(boolean updateTarget)
             return this;
         }
 
+        public TableScanBuilder setNodeId(PlanNodeId id)
```

Comment:
why was this needed ? usually plan matcher tests are not dependant on plan node id

Reply:
it is needed for stats override